### ISSUE TITLE
Add BEL Version to an API endpoint for exposure in BELMgr

### ISSUE THREAD

**Issue by wshayes** [OpenBEL/bel_parser#51]

**Comment by abargnesi** [comment]
I would suggest combining `/api/functions`, `/api/relationships`, and a version endpoint to one route.

- Create a language route (`/api/language...`).
- Consolidate function and relationship into it.
- Add version specification endpoint.

**Comment by wshayes** [comment]
We need to figure out a strategy for updating the API - versioning?   I'd like a way to coordinately update the OpenBEL API without having to do a concurrent release of the BELMgr at the same time - as we move forward with other dependent applications, this problem is going to become more urgent.  I also want to change the name of /api/evidences to /api/nanopubs.

**Comment by abargnesi** [comment]
I definitely understand the API version strategy, but it's a separate issue. Since we're nearing the next major version release it's a good opportunity to get `/api/language` correct now. I don't mind updating the routes across the API, BELMgr, and documentation once this is complete.

**Comment by wshayes** [comment]
Re-worked the language routes as suggested.  Tony added the version api endpoint, but I am still working out a bug where it always responds with the text/plain result rather than the requested JSON result as requested.
